Core/Src/main.c

```diff
@@ -44,6 +46,20 @@
 /* Private variables ---------------------------------------------------------*/
 
 /* USER CODE BEGIN PV */
+	const uint8_t FIRST_BYTE = 0b00000001; //send start bit
+	const uint8_t SECOND_BYTE = 0b10000000; //configure channel select CH0
+	const uint8_t THIRD_BYTE = 0b00000000; //all don't cares to get back 8bit register
+
+	const float COUNTER_PERIOD = 65535;
+	const float FIVE_PERCENT = 0.05;
+	const float FIVE_PERCENT_DUTY_CYCLE = COUNTER_PERIOD * FIVE_PERCENT;
+	const float MAX_BITS = 1024;
+	const float ADC_RATIO = FIVE_PERCENT_DUTY_CYCLE / MAX_BITS;
+
+	const int TOTAL_BITS = 6;
+	const int THREE = 3; //Binary = 11 to select first 2 bits
+	const int EIGHT = 8; //For left shifting by 8 bits
+
```

Comment:
Change to uint8_t because it is explicit in its size and value (being greater than 0)